File: lua/refactoring/tests/refactor/106/lua/with-comments/extract.expected.lua

local function foo_bar(a, test, test_other)
    for idx = test - 1, test_other do
        print(idx, a)
    end
end


-- This is a comment
-- comments are fun
-- wow!
function simple_function(a)
    local test = 1
    local test_other = 11
    foo_bar(a, test, test_other)
end

File: lua/refactoring/tests/refactor/106/c/with-comments/extract.expected.c
#include <stdio.h>

void foo_bar(INSERT_PARAM_TYPE a, INSERT_PARAM_TYPE test, INSERT_PARAM_TYPE test_other) {
      for (int idx = test - 1; idx < test_other; idx++) {
    printf("%d %d", idx, a);
  }
}


/*
 * This is a comment
 * comments are fun
 * wow!
 */
void simple_function(int a) {
  int test = 1, test_other = 1;

foo_bar(a, test, test_other);

}
